src/System.Text.Utf8String/System/Text/Utf8Span.cs

```diff
@@ -508,6 +513,17 @@ private static int LastIndexOf(ReadOnlySpan<byte> buffer, byte value)
             }
             return -1;
         }
+
+        private int Compare(ReadOnlySpan<byte> left, ReadOnlySpan<byte> right)
```

Comment:
I know you said this isn't optimized yet, but can you look at reusing some of https://github.com/dotnet/coreclr/blob/8ec9b34df095df032e37b4c20a7359e9df7ae0ce/src/mscorlib/src/System/String.Comparison.cs#L222

Reply:
I filed issue https://github.com/dotnet/corefx/issues/24880 and added comment to borrow from this implementation.

Reply:
Thanks